egs/babel/s5b/local/lonestar.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
```

Comment:
Some external libraries (like pylauncher in this case) were refactored or extended from Python 2 to 3, but were not backported. This allows the Python3 APIs of these libraries to be compatible with Python2. Since the original code was in Python2 anyway, we are anyway just using the functions which were available then. So this does not affect anything. I'll remove these imports.